napari/_qt/widgets/_tests/test_qt_dock_widget.py

```diff
@@ -139,3 +141,15 @@ def test_adding_stretch(make_napari_viewer):
     dw = viewer.window.add_dock_widget(widg, area='bottom')
     assert widg.layout().count() == 1
     dw.close()
+
+
+def test_combine_widgets_error():
+    """Check error raised when combining widgets with invalid types."""
+    with pytest.raises(
+        TypeError, match='"widgets" must be a QWidget, a magicgui'
+    ):
+        combine_widgets(
+            [
+                'string',
+            ]
+        )
```

Comment:
This formatting made my head hurt 😂 and it's because of the comma:

```suggestion
        combine_widgets(['string'])
```

Reply:
:exploding_head: omg that's why. Let me fix it.